tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "strictNullChecks": true,
     "strictFunctionTypes": true,
+    "skipLibCheck": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
     "noUnusedLocals": true,
```

Comment:
Can I ask you to remove `removeComments` as well? Should not be used for the libraries.

Reply:
Fixed